### PR TITLE
[No QA] Add PR template to this library

### PR DESCRIPTION
**Pull request by marcaaron**
Adding a minimal PR template to this library to help organize.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+GH_LINK
+
+### Automated Tests
+<!---
+Most changes to Onyx should have an accompanying tests. Describe the tests you added or if no tests were added an explanation about why one was not needed.
+--->
+
+### Linked PRs
+<!---
+Please include links to any PRs where this change will be implemented.
```
Comment (iwiznia) [inline .github/PULL_REQUEST_TEMPLATE.md:17]:
```suggestion
Please include links to any PRs where this change will start being used
```
?

**Review by iwiznia — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (marcaaron) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: It does sound a little weird maybe

>Please include links to any PRs in repos that update the `package.json` version.

**Review by iwiznia — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+### Details
+<Explanation of the change or anything fishy that is going on>
+
+### Related Issues
+<Please replace GH_LINK with the link to the GitHub issue this Pull Request is related to>
+GH_LINK
+
+### Automated Tests
+<!---
+Most changes to Onyx should have an accompanying tests. Describe the tests you added or if no tests were added an explanation about why one was not needed.
```
Comment (madmax330) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: should be `have accompanying tests` or `have an accompanying test`

**Comment by marcaaron** [comment]
Updated

**Review by iwiznia — APPROVED** [review]

**Pull request merged**